chart_review/cli.py

```diff
@@ -5,6 +5,7 @@
 
 from chart_review import cohort, config
 from chart_review.commands.accuracy import accuracy
+from chart_review.commands.info import info
```

Comment:
I'm open to feedback on `info`. Right now it basically shows some pieces of the final/computed config. But it might grow a bit.

A possible future thing we may want to add to this subcommand (probably behind a flag? or maybe its own subcommand?) is the various id mappings for notes?

Reply:
i think `info` is fine - `status` is the only alternate i could think of that i thought was reasonable, and i like `info` more than that.